src/content/docs/tutorial-kubernetes-learn/tutorial-k8s-layers.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: Simplify kubernetes layers with New Relic
+---
+
+import k8sDiagramContext from 'images/tutorials_diagram_kubernetes-overview.webp'
+
+import tutorialOverviewdashboard from 'images/tutorial_screenshot-full_overview-dashboard.webp'
+
+import tutorialOverviewDeployments from 'images/tutorial_screenshot-full_overview-deployments.webp'
+
+import tutorialOverviewFailed from 'images/tutorial_screenshot-crop_failed-pods.webp'
+
+import tutorialAPMOverview from 'images/tutorial_screenshot-full_apmK8s.webp'
+
+import tutorialAPMPerformance from 'images/tutorial_screenshot-full_apm-k8s-performance.webp'
+
+Understanding the inner workings of your Kubernetes system, specifically how all the different functions of the system come together, is inherently complex. Let's review what makes up a Kubernetes system and then dive into how New Relic can help monitor and understand each part.
+
+## Break it apart to understand
```

Comment:
What if I already know how Kubernetes works? What if I only want to know how New Relic works with Kubernetes to solve *my* problems?